src/XrdUtils.cmake

```diff
@@ -67,6 +67,25 @@ add_library(
   XrdTls/XrdTlsNotaryUtils.icc  XrdTls/XrdTlsNotaryUtils.hh
   XrdTls/XrdTlsPeerCerts.cc     XrdTls/XrdTlsPeerCerts.hh
   XrdTls/XrdTlsSocket.cc        XrdTls/XrdTlsSocket.hh
+  XrdTls/XrdTlsTempCA.cc        XrdTls/XrdTlsTempCA.hh
+
+  #-----------------------------------------------------------------------------
+  # XrdCrypto: linking against a few functions that are useful for XrdTls; avoids
+  # linking against all of libXrdCryptossl in XrdUtils
```

Comment:
OK, I see the problem. I really don't know why this was made a plug-in to begin with. It would probably take too much time to figure out the history. My presumption is that at the time gsi was written there could be a choice of which crypto functions could be used. Practically, I don't think there  is a choice any more so it needn't be a plug-in. That said, not making it a plugin would then require changes in gsi, sigh. So, let's live with it this way. Note that we will have to mollify EPEL in that these files will be compiled twice, once for the plugin and once for XrdUtils.so, ughhh.

Reply:
> My presumption is that at the time gsi was written there could be a choice of which crypto functions could be used

My understanding is this was written "just in case" some other backend (like `libnss`) would come along and be a pluggable replacement to OpenSSL.  Personally, I think making a generic plugin mechanism "just in case" is more effort than just writing two implementations... anyhow, past is past.

I don't think there's any problem in EPEL to compile twice here -- with EPEL, the usual concern is "vendoring" where you compile in a 3rd party package instead of depending on it as a separate RPM.

Since there's nothing to do currently, I'll just resolve this thread.